praw/models/reddit/live.py

```diff
@@ -37,3 +59,96 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
 
     def _info_path(self):
         return API_PATH['liveabout'].format(id=self.id)
+
+    @property
+    def fullname(self):
+        """Return ``name`` of :class:`.LiveThread`."""
+        return 'LiveUpdateEvent_' + self.id
+
+    def updates(self, **generator_kwargs):
+        """Return a :class:`.ListingGenerator` yields :class:`.LiveUpdate` s.
+
+        :param generator_kwargs: keyword arguments passed to
+            :class:`.ListingGenerator` constructor.
+        :returns: A :class:`.ListingGenerator` object which yields
+            :class:`.LiveUpdate` object.
+        """
+        path = API_PATH['live_updates'].format(id=self.id)
```

Comment:
I realize this is a path, though for consistency could you call it `url`? If you want to go through and replace all instances of `url` in such cases with `path` in a separate PR, I would be okay with that.